babel.config.js

```diff
@@ -26,7 +26,6 @@ module.exports = {
       ],
     },
     production: {
-      only: ["src"],
```

Comment:
this was causing a build error when running `yarn build` with `NODE_ENV=production` Since we don't have a `src` directory, I assume it just means that this whole section previously wasn't being used